apps/src/gamelab/GameLabP5.js

```diff
@@ -0,0 +1,435 @@
+'use strict';
```

Comment:
Shameless plug: I like to add [these stricter linting rules](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/EventSandboxer.js#L2) to new files, as we gradually work toward making them global linting rules. :grin: